src/lib/ecore_imf/meson.build

```diff
@@ -4,6 +4,7 @@ ecore_imf_ext_deps = [buildsystem_simple]
 
 ecore_imf_header_src = [
    'Ecore_IMF.h'
+   'ecore_imf_api.h'
```

Comment:
There are other `,` missing

Reply:
I'd go ahead and add trailing comma to all the lists you're touching.

Reply:
yah, just checked it. I hope I didn't miss anyone, I will wait for Travis answer.